### ISSUE TITLE
Closed caption

### ISSUE THREAD

**Issue by texas2010** [FreeCodeCamp/hikes#160]
All videos need to have a closed caption.

**Comment by BerkeleyTrue** [comment]
How is different from the transcript?

**Comment by matthras** [comment]
dwd2010 probably means real-time captions on the videos, which require formatting the transcript to have timestamps for each line in the video. It does have a convenience factor for the viewers.

It's additional work, but there should be software that makes it easier.

**Comment by QuincyLarson** [comment]
We are including this topic in our core curriculum so I'm closing the video challenge issue.
